portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/SegmentationCreateSegment.testcase

```diff
@@ -2755,7 +2830,9 @@ definition {
 				locator1 = "Segmentation#CARD_HEADING",
 				panelHeader = "Scope");
 
-			AssertNotVisible(locator1 = "Segmentation#SCOPE_CARD_SITE");
+			AssertNotVisible(
```

Comment:
Why did you add this change in this test?

Reply:
by the way the path was made it was finding the message above the site name and not the site name, I refactored the path and needed to correct the test that uses this path. 

But I forgot that the tests have the feature flag property! I will remove this commit but when the feature flag is removed this test will probably need a fix!